src/components/pages/GenerationPage/GeneratePetitionModal/GeneratePetitionModal.js

```diff
@@ -22,6 +22,12 @@ const GeneratePetitionModal = ({ closeModal, isVisible }) => {
   const _buildPetition = () => {
     return {
       petition: petition.pk,
+      name_petitioner: batch.label,
```

Comment:
I think we actually want to allow the attorney to override the name. It can default to ``batch.label`` when the page loads, but a field should be presented to the user to override it. The backend extracts batch.label from one of many possible records and an attorney may need to adjust it.

Reply:
Ah i see. That should be a quick fix